README.md

```diff
@@ -162,8 +162,7 @@ is how you can run them:
 ```sh
 mkdir build && cd build
 cmake ..
-make python_install
-make check
+make check-ut
```

Comment:
Was this change intended?

Reply:
This hunk is not related to this PR (commit message) looked odd to be here; but I'm practicing to avoid nit-picking in reviews :)

But the change itself is [sound](https://github.com/goToMain/libosdp/blob/master/tests/unit-tests/CMakeLists.txt#L45).